.github/workflows/build.yml

```diff
@@ -47,6 +47,7 @@ jobs:
               -DMACOS_BUNDLE=ON
               -DMODPLUG=ON
               -DQT6=ON
+              -DQML=OFF
```

Comment:
Hm, this PR seems to have broken the build with QML on macOS (which is the default) and since we've disabled it in CI, some issues seem to have flown under the radar here... I'll have a quick look at how easy that would be to fix